documents/ogc-submission.adoc

```diff
@@ -47,38 +47,37 @@ Once approved, the Community Standard Work Item defined by this document is vali
 
 == Overview of proposed submission
 
-openEO defines an open application programming interface (API) that connects clients in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
+openEO specifies an open application programming interface (API) that connects clients in languages such as R, Python, and JavaScript to big Earth observation cloud back-ends in a simple and unified way.
 
-The openEO specifications aim at increasing the interoperability of big EO data processing of satellite imagery in the cloud. Implementations of the openEO API can be used to add an interoperability layer on top of existing services.
-Multiple organizations agreed to define and implement such an API. Its development was driven by the need to overcome the challenges associated with different tools, APIs, and data formats in geospatial technology. openEO was developed bottom-up and each version was backed by implementations.
+The openEO specification aims at increasing the interoperability of big EO data processing of satellite imagery in the cloud. Implementations of openEO can be used to add an interoperability layer on top of existing services.
+Its development was driven by the need to overcome the challenges associated with different tools, APIs, and data formats in geospatial technology. openEO was developed bottom-up and each version of the specification was backed by implementations.
 
-The primary use case for developing openEO was to simplify and unify the data processing using a common API specification with a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints supporting comparing and reproducing processing results more easily between different providers.
+The primary use case for specifying openEO was to simplify and unify the data processing using a common API and a set of pre-defined processes. As such, users can still work in their favored programming language without worrying about data organization and pre-processing. Users can avoid vendor lock-in as the generated process descriptions can be executed at multiple provider endpoints supporting comparing and reproducing processing results more easily between different providers.
 
-openEO is an open source project and the specification consists of two parts:
+The specification consists of two parts:
 * The openEO API, an HTTP API description specified as an OpenAPI document
 * The openEO Processes, a set of pre-defined processes defined following the openEO process description in JSON that is defined in the openEO API specification
```

Comment:
```suggestion
* The openEO Processes, a set of pre-defined processes defined following the openEO Process Description in JSON that is defined in the openEO API specification
```

Reply:
I'm not sure, what do the others think?

Reply:
I'm OK with lowercase - "process description" is neither a well-known name nor a term defined in this document.

```suggestion
* The openEO Processes, a set of requirements for pre-defined processes conforming to the openEO API specification
```